optuna/samplers/_cmaes.py

```diff
@@ -177,6 +180,15 @@ def objective(trial):
                 versions without prior notice. See
                 https://github.com/optuna/optuna/releases/tag/v2.6.0.
 
+        with_margin:
+            If this is :obj:`True`, CMA-ES with margin is used.
```

Comment:
How about adding the paper of CMA-ES with Margin in the following list?

https://github.com/optuna/optuna/blob/23c874764b2be969ebe19c86135614a513ff8ae4/optuna/samplers/_cmaes.py#L77-L91

Reply:
Furthermore, how about adding the description about when Optuna users should use this option?

Reply:
Thanks! I updated the docstring according to your comment.